bcda/api.go

```diff
@@ -307,3 +308,15 @@ func blueButtonMetadata(w http.ResponseWriter, r *http.Request) {
 
 	w.WriteHeader(http.StatusOK)
 }
+
+func metadata(w http.ResponseWriter, r *http.Request) {
+	dt := time.Now()
+
+	scheme := "http"
+	if r.TLS != nil {
+		scheme = "https"
+	}
+	host := fmt.Sprintf("%s://%s", scheme, r.Host)
+	statement := responseutils.CreateCapabilityStatement(dt, "0.1", host)
```

Comment:
Hardcoding version 0.1 ?

Reply:
well its a parameter value that can be changed to whatever we want in the future. its really a placeholder at the moment

Reply:
Yeah, I think the concern is that it is hardcoded in the code, which means we have to change it whenever the version might change.  We'll probably run into this in other tasks as well (see BCDA-317).

Reply:
It's possible to provide a version string at compile time so that we can keep this up to date.

For example:

```
go build -ldflags "-X main.version=$(VERSION_STRING)"
```

And in `package main`, define:

```
var version string
```

Then use `version` where appropriate.

As long as there are no objections, I say we leave this as-is for now. I will add this info to BCDA-317 and note that, once this is incorporated, we should also update the `CapabilityStatement` to use the value of `version`.

Reply:
Works for me!

Reply:
yep yep